### PR TITLE
[WIP] Create network elements by dataframe

### PR DESCRIPTION
**Pull request by yichen88**
Signed-off-by: yichen88 <tang.yichenyves@gmail.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -32,6 +32,13 @@ def test_run_lf(self):
         results = pp.loadflow.run_dc(n, parameters)
         self.assertEqual(1, len(results))
 
+    def test_new_load(self):
+        n = pp.network.create_eurostag_tutorial_example1_network()
+        n.new_load('VLLOAD', 'LOAD2', double_map={"p0" : 1.1, "q0" : 2.1}, str_map={'connectable_bus':'NLOAD', 'bus': 'NLOAD'})
```
Comment (geofjamg) [inline tests/test.py]: Why not doing it using a dataframe update? that would be more consistent with what has already been done.

**Review by geofjamg — CHANGES_REQUESTED** [review]

File: cpp/src/pypowsybl.cpp [hunk 2]
```diff
@@ -418,6 +418,19 @@ void updateNetworkElementsWithStringSeries(void* network, element_type elementTy
                 idPtr.get(), valuePtr.get(), elementCount);
 }
 
+void newElement(void* network, const std::string& id, element_type elementType,
```
Comment (geofjamg) [inline cpp/src/pypowsybl.cpp]: @yichen88 have you tried to create Series from Python side instead of having all theses maps?

Comment (yichen88) [reply cpp/src/pypowsybl.cpp]: When we add elements, we need a `ROW` of data to build an equipment together, we can not do with `COLUMN'` in Series.

**Comment by sylvlecl** [comment]
**About scope of that evolution:** 
we can start with a first evolution with only generators and loads (which already have use cases).

**Then we will have a design question for elements with "nested" elements**, such as transformers and tap changer steps, or shunts and shunts models. They cannot be represented as a single dataframe. A possibility would be to be able to pass multiple dataframes : for example 1 for transformers data and 1 for the corresponding  steps data.

**Comment by sylvlecl** [comment]
We will need to refactor a little how data is provided from python to java, based on the work on #234 .

As for the element updates, we should now transfer the whole dataframe at once.

**Comment by sylvlecl** [comment]
Will be merged with / refactored in #259

**Pull request closed without merging**